keyboards/ergodox/keymaps/narze/keymap.c

```diff
@@ -0,0 +1,852 @@
+        // eeprom_update_byte(EECONFIG_SUPERDUPER_INDEX, _QWERTY);
+      }
+      return false;
+      break;
+    case COLEMAK:
+      if (record->event.pressed) {
+        persistant_default_layer_set(1UL<<_COLEMAK);
+
+        // key_combos[CB_SUPERDUPER].keys = superduper_combos[_COLEMAK];
+        // eeprom_update_byte(EECONFIG_SUPERDUPER_INDEX, _COLEMAK);
+      }
+      return false;
+      break;
+    case QWOC:
+      if (record->event.pressed) {
+        persistant_default_layer_set(1UL<<_QWOC);
+
+        // key_combos[CB_SUPERDUPER].keys = superduper_combos[_QWOC];
+        // eeprom_update_byte(EECONFIG_SUPERDUPER_INDEX, _QWOC);
+      }
+      return false;
+      break;
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case RAISE:
+      if (record->event.pressed) {
+        layer_on(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case SUPER:
+      if (record->event.pressed) {
+        layer_on(_SUPER);
+        update_tri_layer(_SUPER, _DUPER, _SUPERDUPER);
+      } else {
+        layer_off(_SUPER);
+        update_tri_layer(_SUPER, _DUPER, _SUPERDUPER);
+      }
+      return false;
+      break;
+    case DUPER:
+      if (record->event.pressed) {
+        layer_on(_DUPER);
+        update_tri_layer(_SUPER, _DUPER, _SUPERDUPER);
+      } else {
+        layer_off(_DUPER);
+        update_tri_layer(_SUPER, _DUPER, _SUPERDUPER);
+      }
+      return false;
+      break;
+    case BACKLIT:
+      if (record->event.pressed) {
+        register_code(KC_RSFT);
+        #ifdef BACKLIGHT_ENABLE
+          backlight_step();
+        #endif
+      } else {
+        unregister_code(KC_RSFT);
+      }
+      return false;
+      break;
+    case PLOVER:
+      if (record->event.pressed) {
+        layer_off(_RAISE);
+        layer_off(_LOWER);
+        layer_off(_ADJUST);
+        layer_on(_PLOVER);
+        if (!eeconfig_is_enabled()) {
+            eeconfig_init();
+        }
+        keymap_config.raw = eeconfig_read_keymap();
+        keymap_config.nkro = 1;
+        eeconfig_update_keymap(keymap_config.raw);
+      }
+      return false;
+      break;
+    case EXT_PLV:
+      if (record->event.pressed) {
+        layer_off(_PLOVER);
+      }
+      return false;
+      break;
+    case SDTOGG:
+      if (record->event.pressed) {
+        // superduper_enabled = !superduper_enabled;
+
+        // if (superduper_enabled) {
+        //   set_superduper_key_combos();
+        // } else {
+        //   clear_superduper_key_combos();
+        // }
+      }
+      return false;
+      break;
+    case EPRM:
```

Comment:
This is no longer needed, and is a core keycode. 

https://docs.qmk.fm/#/quantum_keycodes?id=qmk-keycodes
`EEP_RST`